### DISCUSSION TITLE
Gists Former Revisions

### DISCUSSION THREAD

**Discussion by yurylavrukhin** [community/community#38479] [Other Feature Feedback, Questions, & Ideas]
I wanted to view my former gist version, I went to Revisions tab and found button "View File", but it actually brings the user to the former version of every file on that gist.

Should we rename this button to "View gist"?

Steps to reproduce:
1. Go to [demo](https://gist.github.com/yurylavrukhin/fbc3e556185da8a0576e3ab75df373f7/revisions) gist revisions tab 
2. Click "View file"
<img width="1661" alt="image" src="https://user-images.githubusercontent.com/26966797/200312737-f5c14ae5-3d82-4ddb-8077-0e6fda436964.png">

3. You will be redirected [here](https://gist.github.com/yurylavrukhin/fbc3e556185da8a0576e3ab75df373f7/d2893aa29b5541713ebb717b6665807f9696e3b6), where you'll see all files on that gist, not a single file as the button says

**Comment by ossanna16** [comment]
Hi there @yurylavrukhin and welcome to our community! Thank you for the feedback!
